docs/eventing/brokers/broker-types/kafka-broker/configuring-kafka-features.md

```diff
@@ -0,0 +1,206 @@
+# Configuring Kafka Features
+
+There are many different configuration options for how Knative Eventing and the Kafka Broker interact with Apache Kafka.
```

Comment:
```suggestion
There are many different configuration options for how Knative Eventing and the Knaitve Broker for Apache Kafka interact with the Apache Kafka clusters.
```

Reply:
Can we reword this so that we can link to the experimental features page that provides the context of this?